lib/MetaCPAN/Server/Model/CPAN.pm

```diff
@@ -11,9 +11,9 @@ extends 'Catalyst::Model';
 with 'CatalystX::Component::Traits';
 
 has esx_model => (
-    is         => 'ro',
-    lazy_build => 1,
-    handles    => ['es'],
+    is      => 'ro',
+    handles => ['es'],
+    default => sub { MetaCPAN::Model->new( es => $_[0]->servers ) },
```

Comment:
It's accessed in BUILD... didn't seem like something that really needs to be lazy.
What's the benefit of it being lazy?